### ISSUE TITLE
Unicode Error, only in python 2

### ISSUE THREAD

**Issue by abegong** [fivetran/great_expectations#676]
Building documentation in python 2 sometimes results in a unicode error.

I suspect that this is the tip of an iceberg. We may have a fair amount of work to do to button up py2 unicode handling in the renderers.

The data itself is from 538's riddler-pick-lowest data: https://github.com/fivethirtyeight/data/blob/master/riddler-pick-lowest/low_numbers.csv

Here's the directory structure before running `init`
```
.
└── riddler-pick-lowest
    ├── README.md
    └── low_numbers.csv
```

Here are reproduction steps.
```
$ great_expectations init
  _____                _   
 / ____|              | |  
| |  __ _ __ ___  __ _| |_ 
| | |_ | '__/ _ \/ _` | __|
| |__| | | |  __/ (_| | |_ 
 \_____|_|  \___|\__,_|\__|
                           
                           
 ______                      _        _   _                 
|  ____|                    | |      | | (_)                
| |__  __  ___ __   ___  ___| |_ __ _| |_ _  ___  _ __  ___ 
|  __| \ \/ / '_ \ / _ \/ __| __/ _` | __| |/ _ \| '_ \/ __|
| |____ >  <| |_) |  __/ (__| || (_| | |_| | (_) | | | \__ \
|______/_/\_\ .__/ \___|\___|\__\__,_|\__|_|\___/|_| |_|___/
            | |                                             
            |_|                                             


Always know what to expect from your data.

If you're new to Great Expectations, this tutorial is a good place to start:

    https://docs.greatexpectations.io/en/latest/getting_started/cli_init.html?utm_source=cli&utm_medium=init&utm_campaign=0_7_8__develop


Let's add Great Expectations to your project, by scaffolding a new great_expectations directory:

    great_expectations
        ├── great_expectations.yml
        ├── datasources
        ├── expectations
        ├── fixtures
        ├── notebooks
        ├── plugins
        ├── uncommitted
        │   ├── validations
        │   ├── credentials
        │   ├── documentation
        │   └── samples
        └── .gitignore

OK to proceed?
 [Y/n]: 

Done.

========== Datasources ==========

See https://docs.greatexpectations.io/en/latest/core_concepts/datasource.html?utm_source=cli&utm_medium=init&utm_campaign=0_7_8__develop for more information about datasources.


Configure a datasource:
    1. Pandas DataFrame
    2. Relational database (SQL)
    3. Spark DataFrame
    4. Skip datasource configuration
: 1
1

Enter the path of the root directory where the data files are stored.
(The path may be either absolute or relative to current directory.)
: riddler_pick_data
Error: Directory "riddler_pick_data" does not exist.

Enter the path of the root directory where the data files are stored.
(The path may be either absolute or relative to current directory.)
: ./riddler_pick_data
Error: Directory "./riddler_pick_data" does not exist.

Enter the path of the root directory where the data files are stored.
(The path may be either absolute or relative to current directory.)
: riddler-pick-lowest

Give your new data source a short name.
 [riddler-pick-lowest__dir]: 

========== Profiling ==========

Profiling 'riddler-pick-lowest__dir' will create candidate expectations and documentation.

Please note: Profiling is still a beta feature in Great Expectations.  The current profiler will evaluate the entire 
data source (without sampling), which may be very time consuming. 
As a rule of thumb, we recommend starting with data smaller than 100MB.

To learn more about profiling, visit https://docs.greatexpectations.io/en/latest/guides/profiling.html?utm_source=cli&utm_medium=init&utm_campaign=0_7_8__develop.

Found 1 data assets from generator default

Would you like to profile 'riddler-pick-lowest__dir'?
 [Y/n]: 
Profiling 'riddler-pick-lowest__dir' with 'BasicDatasetProfiler'
Profiling all 1 data assets from generator default
	Profiling 'low_numbers'...
            Preparing column 0 of 2: Your Number
            Preparing column 1 of 2: Show Your Work
	14 expectation(s) included in expectation_suite.
	Profiled 2 columns using 3660 rows from low_numbers (0.185 sec)

    Profiled 1 of 1 named data assets, with 3660 total rows and 2 columns in 0.19 seconds.
    Generated, evaluated, and stored 14 candidate Expectations.
    Note: You will need to review and revise Expectations before using them in production.

========== Data Documentation ==========

Great Expectations can create data documentation from the data you just profiled.

To learn more: https://docs.greatexpectations.io/en/latest/guides/data_documentation.html?utm_source=cli&utm_medium=init&utm_campaign=0_7_8__develop

Build HTML documentation? [Y/n]: 

Building documentation...
        Rendering profiling for data asset riddler-pick-lowest__dir/default/low_numbers
Traceback (most recent call last):
  File "/Users/abe/anaconda2/bin/great_expectations", line 11, in <module>
    load_entry_point('great-expectations', 'console_scripts', 'great_expectations')()
  File "/Users/abe/Documents/superconductive/tools/great_expectations/great_expectations/cli/cli.py", line 327, in main
    cli()
  File "/Users/abe/anaconda2/lib/python2.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/Users/abe/anaconda2/lib/python2.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/Users/abe/anaconda2/lib/python2.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/abe/anaconda2/lib/python2.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/abe/anaconda2/lib/python2.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/Users/abe/Documents/superconductive/tools/great_expectations/great_expectations/cli/cli.py", line 209, in init
    profile_datasource(context, data_source_name)
  File "/Users/abe/Documents/superconductive/tools/great_expectations/great_expectations/cli/datasource.py", line 289, in profile_datasource
    build_documentation(context)
  File "/Users/abe/Documents/superconductive/tools/great_expectations/great_expectations/cli/datasource.py", line 308, in build_documentation
    index_page_locator_infos = context.build_data_documentation(site_names=site_names, data_asset_name=data_asset_name)
  File "/Users/abe/Documents/superconductive/tools/great_expectations/great_expectations/data_context/data_context.py", line 1369, in build_data_documentation
    index_page_locator_info = SiteBuilder.build(self, complete_site_config, specified_data_asset_name=data_asset_name)
  File "/Users/abe/Documents/superconductive/tools/great_expectations/great_expectations/render/renderer/site_builder.py", line 119, in build
    site_config['profiling_store']['name']
  File "/Users/abe/Documents/superconductive/tools/great_expectations/great_expectations/render/renderer/site_builder.py", line 286, in generate_profiling_section
    model = profiling_renderer_class.render(validation)
  File "/Users/abe/Documents/superconductive/tools/great_expectations/great_expectations/render/renderer/page_renderer.py", line 358, in render
    ) for column in ordered_columns
  File "/Users/abe/Documents/superconductive/tools/great_expectations/great_expectations/render/renderer/column_section_renderer.py", line 65, in render
    content_blocks.append(cls._render_values_set(evrs))
  File "/Users/abe/Documents/superconductive/tools/great_expectations/great_expectations/render/renderer/column_section_renderer.py", line 362, in _render_values_set
    values = [str(v["value"]) for v in partial_unexpected_counts]
UnicodeEncodeError: 'ascii' codec can't encode character u'\xaf' in position 0: ordinal not in range(128)
```

**Comment by jcampbell** [comment]
This is an artifact of a lazy use of str to cast. I'm going to update to a more py2/py3 compatible approach.

**Comment by jcampbell** [comment]
Fixed in #684 for this particular instance, but I'm viewing py2 support as generally low priority here.
